filtering scripts/filtering.py

```diff
@@ -0,0 +1,50 @@
+from pathlib import Path
+from mdframe.reader import run, Config
+from filtering_helper import dataframe_check
+
+def filter_files(nutrition_subgroup : str = None, food_type : str = None, description : str = None, weight : float = None, 
+              unit : str = None, ingredients : list = None, project_name : str = None, rgbd_file_names : list = None, 
```

Comment:
The type annotations typically have the colon stuck to the variable name, e.g. `unit: str` not `unit : str`. Also, if the default value for an argument is set to `None` the correct way of specifying the type annotation is: `unit: str | None = None`. That syntax only works with Python 3.10 and later (which is fine for this project), in older versions of Python you might see something like `unit: Optional[str] = None`.